src/service/video-manager-impl.js

```diff
@@ -577,11 +595,11 @@ class VideoEntry {
     analyticsEvent(this, VideoAnalyticsEvents.CUSTOM, prefixedVars);
   }
 
-  /** Listens for signals to delegate autoplay to a different module. */
-  listenForAutoplayDelegation_() {
+  /** Listens for signals to delegate playback to a different module. */
+  listenForPlaybackDelegation_() {
     const signals = this.video.signals();
-    signals.whenSignal(VideoServiceSignals.AUTOPLAY_DELEGATED).then(() => {
-      this.allowAutoplay_ = false;
+    signals.whenSignal(VideoServiceSignals.PLAYBACK_DELEGATED).then(() => {
+      this.allowPlayback_ = false;
```

Comment:
nit: `allowPlayback_` might incorrectly be interpreted as blocking playback somehow. Maybe `managePlayback_`